rpcclient/infrastructure.go

```diff
@@ -932,13 +932,13 @@ func (c *Client) sendCmd(cmd interface{}) chan *response {
 	return responseChan
 }
 
-// sendCmdAndWait sends the passed command to the associated server, waits
+// SendCmdAndWait sends the passed command to the associated server, waits
 // for the reply, and returns the result from it.  It will return the error
 // field in the reply if there is one.
-func (c *Client) sendCmdAndWait(cmd interface{}) (interface{}, error) {
+func (c *Client) SendCmdAndWait(cmd interface{}) (interface{}, error) {
 	// Marshal the command to JSON-RPC, send it to the connected server, and
 	// wait for a response on the returned channel.
-	return receiveFuture(c.sendCmd(cmd))
+	return ReceiveFuture(c.SendCmd(cmd))
```

Comment:
What do you think of deleting this method? It is only used a single place. It can be replaced by inling the contents here. That way there's one less public symbol exported, which is nice.

Reply:
Whoops, making that one public was unintentional; seems I was a tad sloppy while rebasing.  Reverted it to private; thanks for catching that.